demos/ros_examples/scripts/joint_space_velocity_control.cpp

```diff
@@ -26,8 +26,8 @@ class RobotInterface {
 
   explicit RobotInterface(ros::NodeHandle* node_handle, const std::string& robot_name, const std::string& urdf_path) :
       robot_(robot_name, urdf_path) {
-    subscriber_ = node_handle->subscribe("/joint_states", 10, &RobotInterface::robot_state_callback, this);
-    publisher_ = node_handle->advertise<std_msgs::Float64MultiArray>("/velocity_controller/command", 10, false);
+    subscriber_ = node_handle->subscribe("joint_states", 10, &RobotInterface::robot_state_callback, this);
```

Comment:
You removed the `/` because of the namespace? Is it because it automatically adds one?

Reply:
If you put the `/`, it will be a global topic, i.e. the full name of the topic is always `/joint_states`. Without the `/` it will be in the namespace of the node, i.e. here `/robot_name/joint_states`

Reply:
oh yeah I remember that now. It does not work like this in ROS2. Alright makes sense